assets/js/base/components/text-input/validated-text-input.tsx

```diff
@@ -91,15 +113,19 @@ const ValidatedTextInput = ( {
 	// Remove validation errors when unmounted.
 	useEffect( () => {
 		return () => {
-			clearValidationError( errorId );
+			clearValidationError( errorIdString );
 		};
-	}, [ clearValidationError, errorId ] );
+	}, [ clearValidationError, errorIdString ] );
 
-	const errorMessage = getValidationError( errorId ) || {};
+	// TODO - When useValidationContext is converted to TypeScript, remove this cast and use the correct type.
```

Comment:
In the past we unified all "to do" comments to use this form: `@todo`. That makes it easier to search them in the codebase. :slightly_smiling_face: